cmake/json_opts.cmake

```diff
@@ -0,0 +1,73 @@
+include(CMakeDependentOption)
+include(GNUInstallDirs)
+
+#############################################################################
+# test options
+#############################################################################
+
+# VERSION_GREATER_EQUAL is not available in CMake 3.1
+if(${JSON_MAIN_PROJECT} AND (${CMAKE_VERSION} VERSION_EQUAL 3.13 OR ${CMAKE_VERSION} VERSION_GREATER 3.13))
```

Comment:
Don't expand variables.

Reply:
After some testing, I changed my mind. I'll deal with this when I address the shortcomings of the CMake flags test.